.github/workflows/duplicate-app-studio-service-issue.yml

```diff
@@ -0,0 +1,28 @@
+name: duplicate issue check for app-studio-service 
+on:
+  issues:
+    types: [opened]
```

Comment:
I think update if needed?
if user update the title or update the description?

Reply:
Updated. Thanks.